frontend/src/features/whats-new/utils/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { isAfter } from 'date-fns'
+
+import { UserDto } from '~shared/types/user'
+
+import { FeatureUpdateList } from '../FeatureUpdateList'
+
+export interface shouldShowLatestFeatureUpdateNotificationProps {
+  user: UserDto | undefined
+  isLoading: boolean
+}
+
+export const shouldShowLatestFeatureUpdateNotification = ({
+  user,
+  isLoading,
+}: shouldShowLatestFeatureUpdateNotificationProps): boolean => {
+  if (isLoading) {
+    return false
+  }
+  const sortedFeatureUpdateList = FeatureUpdateList.sort(
+    (featureUpdateA, featureUpdateB) =>
+      Date.parse(featureUpdateB.date) - Date.parse(featureUpdateA.date),
+  )
+  const lastSeenUserFeatureUpdateDate = new Date(
+    user?.flags?.lastSeenFeatureUpdateDate ??
+      sortedFeatureUpdateList[sortedFeatureUpdateList.length - 1].date,
+  )
```

Comment:
do we need a default? if there is no last seen feature date i think we can return `true` immediately.